aws_codeseeder/commands/_module_commands.py

```diff
@@ -62,9 +62,14 @@ def deploy_modules(seedkit_name: str, python_modules: List[str]) -> None:
     if not stack_exists:
         LOGGER.warn("Seedkit/Stack does not exist")
         return
+
     domain = stack_outputs.get("CodeArtifactDomain")
     repository = stack_outputs.get("CodeArtifactRepository")
 
+    if domain is None or repository is None:
```

Comment:
should be this `AND`?

Reply:
if either of them is None we need to skip the login. there shouldn't be a case where only one of them is None, but just in case we should use an `or` here.

Reply:
Got it